package.json

```diff
@@ -8,8 +8,8 @@
     "start": "NODE_ENV=production nodemon app.js",
     "start-dev": "cross-env NODE_ENV=development nodemon app.js",
     "format": "prettier-standard \"**/*.{js,md,yml,json}\"",
-    "test": "prettier-standard --lint --check \"**/*.{js,md,yml,json}\" && tape \"test/unit/**/*.js\" | tap-spec",
-    "test:integration": "tape \"test/integration/**/*.js\" | tap-spec",
+    "test": "prettier-standard --lint --check \"**/*.{js,md,yml,json}\" && ava \"test/unit/**/*.js\"",
+    "test:integration": "ava \"test/integration/**/*.js\" -v --timeout=60m",
```

Comment:
How did you generate the snapshot files? Can you add a npm script so we can re-generate them?